src/main/markdown/changelog.md

```diff
@@ -1,5 +1,19 @@
 # Lagom Change Log
 
+
+## Lagom 1.4.2
+
+*Released 22 March 2018*
+
+* [1277](https://github.com/lagom/lagom/issues/1277) Adds Kafka message timestamps to Metadata (Ignasi Marimon-Clos)
+* [1275](https://github.com/lagom/lagom/issues/1275) Document required behaviour of flow to message broker subscriber (Jason Pickens)
+* [1265](https://github.com/lagom/lagom/issues/1265) Build Maven plugin and support for Scala 2.12 only (Tim Moore)
+* [1263](https://github.com/lagom/lagom/issues/1263) Fix Cassandra and JDBC conflicts in TestKit (Tim Moore)
+* [1259](https://github.com/lagom/lagom/issues/1259) Adds more akka-misc serializers on the Migration guides and Mvn archetype (Ignasi Marimon-Clos)
+* [1258](https://github.com/lagom/lagom/issues/1258) Remove unnecessary workarounds in ServiceTestSpec (Tim Moore)
+* [1257](https://github.com/lagom/lagom/issues/1257) Applies suggested solution: use Seq(EmptyTree) for empty traits (Ignasi Marimon-Clos)
+* [1246](https://github.com/lagom/lagom/issues/1246) Removes docs wrt the metrics service and its endpoints (Ignasi Marimon-Clos)
+
```

Comment:
I preferred putting stuff into `1.3.x` rather than here.

Reply:
IMO, we should stop producing this changelog files. We have this information on GitHub already and we are providing the link in the announcement.
I also doubt that someone is really reading the changelog.